dot/rpc/modules/author.go

```diff
@@ -169,16 +169,22 @@ func (cm *AuthorModule) SubmitExtrinsic(r *http.Request, req *Extrinsic, res *Ex
 	vtx := transaction.NewValidTransaction(ext, txv)
 
 	if cm.coreAPI.IsBabeAuthority() {
-		hash, err := cm.txQueueAPI.Push(vtx)
-		if err != nil {
-			log.Trace("[rpc] submitted extrinsic failed to push transaction to queue", "error", err)
+		hash, errQueue := cm.txQueueAPI.Push(vtx)
```

Comment:
fixed: 
```
dot/rpc/modules/author.go:172:9: shadow: declaration of "err" shadows declaration at line 155 (govet)
                hash, err := cm.txQueueAPI.Push(vtx)
                      ^
```

Reply:
it might be nicer to declare `var hash common.Hash` outside of the if statement, then you can leave it as `hash, err = cm.txQueueAPI.Push(vtx)`
not a huge fan of renaming err :p